app/views/content/salaries-and-benefits.md

```diff
@@ -160,6 +160,10 @@ The unqualified teacher salary ranges are:
 | Outer London                             | £24,415 | £35,954 |
 | Inner London                             | £25,831 | £37,362 |
 
+If you're already working as an unqualified teacher and have the right experience, you could be eligible for the [12 week assessment only route to qualified teacher status (QTS)](/train-to-be-a-teacher/assessment-only-route-to-qts).
```

Comment:
```suggestion
If you're already working as an unqualified teacher and have the right experience, you could be eligible for the [assessment only route to qualified teacher status (QTS)](/train-to-be-a-teacher/assessment-only-route-to-qts).
```

Given that our AO page says `up to 12 weeks`, should we remove 12 weeks from this link?